test/e2e/framework/test_context.go

```diff
@@ -154,6 +156,9 @@ type CloudConfig struct {
 	MasterTag         string
 
 	Provider cloudprovider.Interface
+
+	UseKubemarkProvider bool
```

Comment:
Currently the ExternalKubeConfig defaults to the kubeconfig in home directory (mostly because it was convenient for local testing) so it is always specified. But removing the default to reduce the number of flags sounds reasonable.